### DISCUSSION TITLE
CP on server issues since update

### DISCUSSION THREAD

**Discussion by Rainey95** [Courseplay/Courseplay_FS22#2382] [Q&A]
Good afternoon, im inquiring if anyone has had any issues with CP on multiplayer dedicated servers? My dedicated server had issues with it since the update from Giants was released. Has anyone encountered this as well?

**Comment by Tensuko** [comment]
If you would describe what Issue and Error, ppl might be able to answer you and we as well could try to figure out the problem and create a fix for it.

**Reply by Rainey95** [reply]
Sure, it threw a error when i loaded into the server it would kick me out of the server, once it picked up the files to use for CP. I tried to relog multiple times and it continued to do it.

**Reply by Tensuko** [reply]
Sounds like something is completly wrong on your side.
Try to log into the server, if you get Errors again, take your log.txt and attach it here (do not copy and paste the text)
